tests/director/tests/runtime_api_test.go

```diff
@@ -726,6 +729,43 @@ func TestQuerySpecificRuntimeWithCertificate(t *testing.T) {
 	})
 }
 
+func TestRuntimeRegistrationWithIntegrationSystemCredentials(t *testing.T) {
```

Comment:
Do you think it's worth asserting that the `kyma` label is not present in some existing test where runtime is not registered with Integration System?